packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2209,6 +2209,190 @@ describe('plugin-meetings', () => {
           );
           assert.isTrue(myPromiseResolved);
         });
+
+        it('should request floor only after roap transaction is completed', async () => {
```

Comment:
jeez what a complicated test :(

Reply:
The test is simple, just the setup of the mocks is complicated. In the tests for updateShare I've separated the setup into `beforeEach()` so the 2 tests there are simpler. I haven't done it here, because that setup is only used in that 1 single test, although it is very similar to the setup done in the updateShare tests, so maybe this all could have been refactored further, but on the other hand the updateShare/updateMedia functions are all going to be deleted when we move to webrtc-core so I didn't want to spend too much time on these tests.